tests/lib/services/dgg-rolling-chat-cache.test.js

```diff
@@ -1,11 +1,21 @@
 const assert = require('assert');
 const sinon = require('sinon');
 const RollingChatCache = require('../../../lib/services/dgg-rolling-chat-cache');
+const config = require('../../../lib/configuration/prod.config.json');
+const logger = require('../../../lib/services/logger');
+const messageMatchingService = require('../../../lib/services/message-matching');
 
 describe('Chat Cache Test suite', () => {
   describe('Chat Cache Viewer Map Tests', () => {
+    beforeEach(() => {
+      this.mockServices = {
+        logger: logger(config.logger),
+        messageMatching: messageMatchingService,
```

Comment:
We should mock the dependencies of the unit under test. This especially makes a difference for the logger, which pollutes the test output with log messages if it isn't mocked. You can use Sinon's stub API.